docs/organizations/verify-user-permissions.mdx

```diff
@@ -370,8 +370,8 @@ You can pass a `role` the same way you can pass a `permission` in all the exampl
 
 In order to enhance typesafety in your project, you can define a global `ClerkAuthorization` interface, which defines the acceptable values for roles and permissions.
 
-> [!INFO]
-> By default, types related to roles and permissions, such as `OrganizationCustomRoleKey` and `OrganizationCustomPermissionKey`, will be assigned to `string` if `ClerkAuthorization` is not defined. If you define `ClerkAuthorization` in your project, `OrganizationCustomRoleKey` and `OrganizationCustomPermissionKey` will be assigned to the keys of the `ClerkAuthorization` interface.
+> [!NOTE]
+> By default, roles and permission types like `OrganizationCustomRoleKey` and `OrganizationCustomPermissionKey` are assigned `string`. However, if a `ClerkAuthorization` type is defined, it will be utilized instead.
```

Comment:
```suggestion
> By default, roles and permission types, such as `OrganizationCustomRoleKey` and `OrganizationCustomPermissionKey`, are assigned `string`. However, if a `ClerkAuthorization` type is defined, it will be utilized instead.
```

Reply:
Thanks for the review @alexisintech, amazing edits.